proto/ibc/applications/fee/v1/genesis.proto

```diff
@@ -38,19 +38,20 @@ message FeeEnabledChannel {
 // RegisteredPayee contains the relayer address and payee address for a specific channel
 message RegisteredPayee {
   // the relayer address
-  string relayer_address = 1 [(gogoproto.moretags) = "yaml:\"relayer_address\""];
+  string relayer = 1;
   // the payee address
   string payee = 2;
   // unique channel identifier
   string channel_id = 3 [(gogoproto.moretags) = "yaml:\"channel_id\""];
```

Comment:
This is a nit, but should we move the `channel_id` to the first position? Usually we have the `port_id` and `channel_id` in the first positions of the `message`s?